src/main.js

```diff
@@ -230,12 +207,45 @@ function switchScreens(mainView, alternateView) {
   alternateView.classList.toggle('hidden')
 }
 
-// function posterFormToggle() {
-//   document.querySelector('.main-poster').classList.toggle('hidden')
-//   document.querySelector('.poster-form').classList.toggle('hidden')
-// }
+// created new function for adding event listeners to each mini poster and
+// for deleting mini posters
+function deleteMiniPoster() {
+  var miniPosters = document.querySelectorAll('.mini-poster')
+
+  for (var i = 0; i < miniPosters.length; i++) {
+    miniPosters[i].addEventListener('dblclick', function() {
+      var thisPosterID
+
+      if (event.target.id){
+        thisPosterID = event.target.id
+      } else {
+        thisPosterID = event.target.parentElement.id
+      }
 
-// function savedPostersToggle() {
-//   document.querySelector('.main-poster').classList.toggle('hidden')
-//   document.querySelector('.saved-posters').classList.toggle('hidden')
-// }
+      for (var i = 0; i < savedPosters.length; i++) {
+        if (savedPosters[i].id == thisPosterID){
+          savedPosters.splice(i, 1)
+        }
+      }
+
+      if (event.target.classList === 'mini-poster') {
```

Comment:
For this use, I think this is fine. But, could be potential bug since classList can return an array and ['mini-poster', 'cool-class'] would fail this if statement.